Library/Homebrew/cask/lib/hbc/cli/internal_checkurl.rb

```diff
@@ -12,7 +12,7 @@ def self.run(*args)
       end
 
       def self.help
-        "checks for bad Cask URLs"
+        "Checks for bad Cask URLs"
```

Comment:
I'd rather we make all of these lowercase to be consistent with non-internal commands.